prog/install_rhizome.rb

```diff
@@ -37,7 +37,9 @@ class Prog::InstallRhizome < Prog::Base
 
   label def install_gems
     sshable.cmd("bundle install --gemfile common/Gemfile --path common/vendor/bundle")
-    sshable.cmd("bundle install --gemfile #{frame["target_folder"]}/Gemfile --path #{frame["target_folder"]}/vendor/bundle")
+    if File.exist?("#{Config.root}/rhizome/#{frame["target_folder"]}/Gemfile")
+      sshable.cmd("bundle install --gemfile #{frame["target_folder"]}/Gemfile --path #{frame["target_folder"]}/vendor/bundle")
+    end
```

Comment:
Doesn't this do something rather unexpected if the two versions of rhizome mismatch? `File.exist?` is done on the deployed clover, and then the sshable cmd happens remotely assuming a match, which there may not be.

Reply:
In previous label, we put the rhizome on the control plane to the sshable server, so they should be same. Unless you are thinking the scenario where this Prog gets bigger and we introduce new paths to this label.

Would you prefer doing `sshable.cmd("test #{frame["target_folder"]}/Gemfile")`?